manager/manager.go

```diff
@@ -144,8 +144,9 @@ func (rm *realManager) HousekeepModel() {
 
 func (rm *realManager) Housekeep() {
 	for {
-		start := rm.lastSync
-		end := start.Add(rm.resolution)
+		end := rm.lastSync.Add(rm.resolution)
+		// We need only one sample every housekeeping. Issue #638.
+		start := end.Add(-time.Second)
```

Comment:
We rely on cAdvisor's predictable performance as of now. 
To be on the safe side, increasing the window a bit SGTM.

>Also - will 1 sec window work with metrics compaction/deduplication strategy in cAdvisor?
I don't understand this question. Can you elaborate?